backend/api-service/src/.gcloudignore

```diff
@@ -0,0 +1,5 @@
+api_inventory.go
+api.go
+logger.go
+model_*.go
+routers.go
```

Comment:
should `main.go` be added to this list? I don't see it being checked in.

Reply:
main.go is listed in backend/api-service/.gcloudignore

It gets generated one level up from these files (no change from the current directory structure)

Reply:
Ah you're right. In that case, can't we delete this file then?

Reply:
Yes - that makes sense. I've deleted the nested one. Thanks!

Reply:
Thanks!